### DISCUSSION TITLE
New release doesn't work with my roaster: Modbus Communication Error

### DISCUSSION THREAD

**Discussion by mrapino** [artisan-roaster-scope/artisan#1095] [General]
Hello, I have a Fabrica Roaster that I just set up with Artisan. At first, I installed the latest release, and I could not get my roaster's Modbus to communicate with the software. The temp readings were not showing an there was a "Modbus Communication Error" on the screen. I tried "order of operations" debugging, I tried adjusting the port settings, everything.

Finally, out of desperation, I tried downgrading to the previous version and viola, the Modbus communication works and I can see my temperature readings.

I don't know if there was a code change in the software that broke the Fabrica setup, or maybe the new version has a couple of extra settings that need to be set correctly, but it's frustrating.

**Comment by MAKOMO** [comment]
Sorry for your frustration on using community driven open-source software. Such software can only be as strong as the community supporting it. Why not make it better your self and support the team with a bug fix? Maybe you could at least give the team the information needed to investigate your issue, like which "latest release" did you install which failed to work and which "previous version" you installed that worked. Maybe you can also share your Artisan communication log file which might contain hints on what went wrong. Without your support this software might never work better for Fabrica Roasters.

**Reply by mrapino** [reply]
Thanks for your reply. Your tone is so so appreciated too. It makes me feel really welcome in this community. Yes, I realize this is open source software. I'm more than happy to provide more information. I'll be able to add more to this discussion later today when I have some time.
